style.css

```diff
@@ -382,3 +417,173 @@ body {
   color: white;
   cursor: pointer;
 }
+
+.cont-menu-list {
+  display: none;
+}
+
+@media (min-width: 600px) {
```

Comment:
Please write the value for min-width property should in range of 992px - 1100px.